pkg/platform/runtime2/runtime.go

```diff
@@ -47,6 +54,9 @@ func new(target setup.Targeter) (*Runtime, error) {
 
 // New attempts to create a new runtime from local storage.  If it fails with a NeedsUpdateError, Update() needs to be called to update the locally stored runtime.
 func New(target setup.Targeter) (*Runtime, error) {
+	if strings.ToLower(os.Getenv(constants.DisableRuntime)) == "true" {
+		return DisabledRuntime, nil
+	}
```

Comment:
I wanted to have the check for `ACTIVESTATE_DISABLE_RUNTIME` in one place only, so it needs to be here.  I did not want to return `nil`, as then the caller would have to check for that.  With the `DisabledRuntime` we will also get better error messages, in case we mess something up...